style.css

```diff
@@ -0,0 +1,3 @@
+body{
+  backround-color:red
```

Comment:
please always remember to terminate after every line of code in your css

Reply:
Kindly add an explanation of why you requested the review. Remember you are trying to correct here and you want the person you are correcting to understand why you made a particular request.